calculator.rb

```diff
@@ -0,0 +1,75 @@
+
+data = [
+  4,
+  "6",
+  { result: "Deset"}
+]
+
+
+puts "#{data[0]} + #{data[1]} = #{data[2][:result]} "
+
+
+
+
+class Calculator 
+  
+  attr_accessor :number
+  
+  def initialize(number)
+    @number = [number]
```

Comment:
number is already array so you can use just 
`@number = number`
then you will not need iterate 2 times for sum
better naming will be 
```
attr_accessor :list_of_numbers

def initialize(list_of_numbers)
    @list_of_numbers = list_of_numbers

```